cmd/ignite/run/start.go

```diff
@@ -74,12 +76,16 @@ func dialSuccess(vm *ignite.VM, seconds int) error {
 	for i := 0; i < seconds*perSecond; i++ {
 		conn, dialErr := net.DialTimeout("tcp", addr, delay)
 		if conn != nil {
-			defer conn.Close()
+			conn.Close()
```

Comment:
From group review:
This fixes a bug where the connections would stay open as this loop continued.
The defers stack at the end of the function which is not within the loop.